src/ExpressionEvaluator/Core/Source/FunctionResolver/FunctionResolver.cs

```diff
@@ -192,13 +192,15 @@ private static bool ShouldEnable(DkmProcess process)
             }
         }
 
+        private const uint RPC_E_DISCONNECTED = 0x80010108;
         private const uint COR_E_BADIMAGEFORMAT = 0x8007000b;
         private const uint CORDBG_E_MISSING_METADATA = 0x80131c35;
 
         private static bool IsBadOrMissingMetadataException(Exception e)
```

Comment:
This looks pretty much the same as the method in MetadataUtilities.  Can we just have one?